packages/back-end/src/jobs/updateExperimentResults.ts

```diff
@@ -143,17 +143,21 @@ async function updateSingleExperiment(job: UpdateSingleExpJob) {
       metricRegressionAdjustmentStatuses,
     } = await getRegressionAdjustmentInfo(experiment, organization);
 
-    currentSnapshot = await createSnapshot(
+    currentSnapshot = await createSnapshot({
       experiment,
-      null,
-      experiment.phases.length - 1,
       organization,
-      null,
-      false,
-      organization.settings?.statsEngine,
+      phaseIndex: experiment.phases.length - 1,
+      statsEngine: organization.settings?.statsEngine,
       regressionAdjustmentEnabled,
-      metricRegressionAdjustmentStatuses
-    );
+      metricRegressionAdjustmentStatuses,
+      sequentialTestingEnabled:
+        experiment?.sequentialTestingEnabled ??
+        !!organization.settings?.sequentialTestingEnabled,
+      sequentialTestingTuningParameter:
+        experiment?.sequentialTestingTuningParameter ??
+        organization.settings?.sequentialTestingTuningParameter ??
+        5000,
```

Comment:
so sad that we have to put this fallback in so many places. I guess there isn't really another solution unless we focus more on migrations/data quality, right?

Reply:
We're actually planning an overhaul of settings overrides (we're calling it Project Scoped Settings, but it's going to encompass all altitudes of override). Until that's ready, I think we need to leave something like this in place.

That said, we could start moving the numerical defaults (like 5000) into a constants file (or files rather, one for FE and one for BE)

Reply:
Sounds good. I'm all for either moving defaults to that constant file in a follow-up PR or possibly just starting one here and then migrating others later over time.